app/views/instanceAMIs.scala.html

```diff
@@ -1,28 +1,32 @@
-@(ssa: SSA, amisWithUpgrades: List[AMI], amiSSAs: List[(SSA, scala.List[AMI])])
+@(ssa: SSA, amisWithInstances: List[(AMI, List[Instance])], amiSSAs: List[(SSA, List[AMI])], agePercentiles: utils.Percentiles)
```

Comment:
We should probably have a `case class AmiWithInstances(ami: AMI, instances: List[Instance])` rather than a tuple `(AMI, List[Instance])` so we can avoid dealing with the awkward `._1` below. What do you think?